ethers-core/src/utils/geth.rs

```diff
@@ -76,7 +76,17 @@ impl GethInstance {
         &self.data_dir
     }
 
+    /// Takes the stderr contained in the child process.
+    ///
+    /// This leaves a `None` in its place, so calling methods that require a stderr to be present
+    /// will fail if called after this.
+    pub fn stderr(&mut self) -> Result<ChildStderr, GethInstanceError> {
+        self.pid.stderr.take().ok_or(GethInstanceError::NoStderr)
```

Comment:
could also do an Option but since we already have `NoStdErr`, let's use this result